### PR TITLE
Fixes an issue that could not search on zone select

### PR DESCRIPTION
**Pull request by utchoang**
@rhtyd cc @svenvogel I found 1 issue: We can't search on zone select input in creating/upload templates/isos. I've fixed it. Thanks.
* Before:

*After:

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan package

File: src/views/image/RegisterOrUploadIso.vue [hunk 1]
```diff
@@ -90,14 +90,7 @@
...
             <a-select-option :value="opt.id" v-for="opt in zones" :key="opt.id">
-              <div v-if="currentForm === 'Upload'">
-                <div v-if="opt.name !== $t('label.all.zone')">
-                  {{ opt.name || opt.description }}
-                </div>
-              </div>
-              <div v-else>
-                {{ opt.name || opt.description }}
-              </div>
+              {{ opt.name || opt.description }}
```
[the remark this answers is not in the document]
Comment (utchoang) [reply src/views/image/RegisterOrUploadIso.vue:93]: @rhtyd All zone not showing on upload templates/isos.

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by davidjumani — APPROVED** [review]
LGTM. Thanks @utchoang !

**Comment by davidjumani** [comment]
@rhtyd Good to go!

**Pull request merged**